### ISSUE TITLE
[WinForms] Deinitialization of CoreWebView2?

### ISSUE THREAD

**Issue by kirsan31** [MicrosoftEdge/WebView2Feedback#588]
`WebView2` is a visual control, that belong to `Form`  and have some layout properties. When user open form, `CoreWebView2` is not initialized. When user want to see something from web, we need to init `CoreWebView2` - ok. But after user end his work, we need to revert `WebView2` to it's original state - not initialized (due to heavy resources usage). Can we something _deinit_ `CoreWebView2` without Dispose whole control? Dispose only `CoreWebView2`?
For now only way I see is remove it from controls, Dispose it, than create new one, add it to controls and apply all layout logic again...

[AB#30102318](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/30102318)

**Comment by champnic** [comment]
Currently there isn't a good way to do this. You can navigate to "about:blank" and hide the control, which will allow it to free some resources and wind down some processes, but it won't be zero. Could that work for you?

**Comment by kirsan31** [comment]
@champnic Thank you. Sorry, but no :( Even after initialization, it's too much for us:
![image](https://user-images.githubusercontent.com/17767561/97964902-911b0b00-1dca-11eb-8037-96d442c98aa4.png)

Overall, I think if you have Init that it wood be nice to have DEinit? May be convert this issue to _Feature / API request_ and i will edit it - what do you think?

**Comment by champnic** [comment]
Ya those values look like what I'd expect. I think a deinit is a great idea, I've added an item on our backlog. Until then, disposing and recreating the control is probably your best bet. Depending on the state that you are losing each time, you may consider a wrapper around the WebView2 that maintains some of the layout, which the WebView2 gets recreated.

**Comment by champnic** [comment]
Hey @kirsan31, this item was discussed with the team and we won't be doing more work here. We're going to be introducing a ["Freeze/Unfreeze" type of API](https://github.com/MicrosoftEdge/WebView2Feedback/pull/621/files?short_path=1568593#diff-156859382ecd97176e185f076ccc75f4b8dddc41a90a93c4f436bf0bc6f8c315) soon that can improve resource usage while using WebView2. Otherwise, if you want to completely revert the control, you will need to Dispose the WebView2, and then recreate/reinitialize it when needed. Thanks!

**Comment by champnic** [comment]
We took another look at the scenario, and believe this may also be useful when the browser process crashes to enable easier crash recovery for the end dev. That scenario also has a workaround (using the ProcessFailed event) but this work would make that simpler. Related: #774

**Comment by kirsan31** [comment]
@champnic 
Do you mean same logic mentioned [here](https://github.com/MicrosoftEdge/WebView2Feedback/issues/774#issuecomment-765802654)?  Simple call `Deinitialization` after `ProcessFailed`?

**Comment by champnic** [comment]
Yes - our controls would themselves listen to `ProcessFailed` and deinitialize, allowing the dev to easily recover the webview if needed.
